assets/section-main-product.css

```diff
@@ -595,15 +604,19 @@ a.product__text {
   .grid__item.product__media-item--full {
     width: 100%;
   }
+
+  .product--columns .product__media-item:not(.product__media-item--variant) {
+    max-width: calc(50% - var(--grid-desktop-horizontal-spacing) / 2);
+  }
```

Comment:
> And then unrelated to this PR, small bug which seem pretty "niche": [cart notification + media button overlaying one another on hover](https://screenshot.click/10-25-nz2or-raawo.mp4)

Took me a while to notice that you were using Safari and equally as long to replicate it 😉 
The last part of your video helped though. I wasn't able to replicate it Chrome or Firefox so I don't think its something we should address, at least not in this PR. Also, with the changes I'm adding to reduce the icon size when selecting 2 columns + L/M/S media sizes, it'll make it even more difficult to replicate this.

Reply:
> One potential thing could be to edit the button size on video and 3D models to show up like they do on thumbnails maybe when we're using the 2 columns layout:

That's a good point. And I agree reducing the icon size won't obscure the image too much when choosing different media sizes on desktop (with 2 columns enabled) but I'm not sure if we should mimic how these icons look with the thumbnail or thumbnail slider layouts. 

I'm not super invested in this, but let me know your thoughts: https://github.com/Shopify/dawn/pull/1766/commits/2b83ac78db4cd4d122aa7d0f9758a49dead79005

Reply:
> Here to follow up on this we will need to update the image sizes we're setting in `srcset` and `sizes` of the `product-media.liquid` file otherwise we're outputting images that are much bigger than needed.
> 
> Which is an interesting point actually. That using a two column layout should be helpful in terms of performance compared to a stacked layout once we updates the sizes 🤔 👍

As per our chat, I made the following improvements to the `product-thumbnail` snippet + chose to use the newer, [image_tag](https://shopify.dev/api/liquid/filters#image_tag) but only for this part in the snippet:

https://github.com/Shopify/dawn/blob/b10202cf32cb930607de4b0611cdde8d2bb24b29/snippets/product-thumbnail.liquid#L92-L111

I think it makes sense to only change this (for the scope of this PR) and if we decide that this is a good performance improvement, then we can tackle them in a follow up 👍🏼 

Changes added here: https://github.com/Shopify/dawn/pull/1766/commits/963e694cbaca8c78bcd5074ee20b7954e2b3d5ad

Reply:
I decided to remove the `width` and `height` that was previously being set here:

https://github.com/Shopify/dawn/commit/963e694cbaca8c78bcd5074ee20b7954e2b3d5ad#diff-5c32c3208fa9015083686af15f70ccb19e704943aa988dffaf3184cefbd1f198L108-L109

The main reason I did this was because the performance was slightly better but if I'm missing something on whether this needs to remain, please let me know 👍🏼 

**With** `width` and `height` being set:

<img src="https://user-images.githubusercontent.com/28404165/184247531-e2d59c3d-db0b-4070-8cfb-10bec7fc4034.png" width="400" height="auto">

**Without** `width` and `height` being set:

<img src="https://user-images.githubusercontent.com/28404165/184247763-9471b637-8d1f-4d5d-81d6-8f85e39794b6.png" width="400" height="auto">